src/utilities.cpp

```diff
@@ -59,7 +59,7 @@ void pasteFilesFromClipboard(FmPath* destPath, QWidget* parent) {
   if(data->hasFormat("x-special/gnome-copied-files")) {
     // Gnome, LXDE, and XFCE
     QByteArray gnomeData = data->data("x-special/gnome-copied-files");
-    char* pdata = gnomeData.data();
+    const char* pdata = gnomeData.constData();
```

Comment:
Yes, gcc. See [here](http://www.cplusplus.com/reference/cstring/strchr/). It's logical that either both of them or none of them is const.